chain-impl-mockchain/benches/tally.rs

```diff
@@ -124,12 +125,13 @@ fn tally_benchmark(
             }
 
             let mut private_voter = controller.wallet(&alias).unwrap();
+            let choice = Choice::new(votes_distribution.sample(&mut rng) as u8);
```

Comment:
same here.
We also previously used choices 1/2, while I think this results in choices 0/1 being used. Don't really now why we used 1/2 though, 0/1 seems more sensible to me.

Reply:
> We also previously used choices 1/2, while I think this results in choices 0/1 being used

This doesn't really matter for a benchmark.